Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG BUILD_FROM
+FROM ${BUILD_FROM}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always tag the version of an image explicitly](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5223963)